hcl2template/testdata/variables/basic.pkr.hcl

```diff
@@ -4,3 +4,34 @@ variables {
     my_secret = "foo"
     image_name = "foo-image-{{user `my_secret`}}"
 }
+
+variable "image_id" {
+  type = string
+  default = "image-id-default"
+}
+
+variable "port" {
+  type = number
+  default = 42
+}
+
+variable "availability_zone_names" {
+  type    = list(string)
+  default = ["us-west-1a"]
+  description = <<POTATO
+Describing is awesome ;D
+POTATO
+}
+
+variable "super_secret_password" {
+  type     = string
+  sensible = true
+description = <<IMSENSIBLE
+Handle with care plz
+IMSENSIBLE
```

Comment:
Hello, I have discovered how the "heredoc" format works and I made a stupid joke. Yes, this works !